### ISSUE TITLE
dataset include different sizes

### ISSUE THREAD

**Issue by remember00000** [lllyasviel/ControlNet#667]
Can the model only accept that the input images are all the same size? My data set is guaranteed to be a multiple of 64 in width and height (obtained from the canny preprocessing codes in this repository), but an error is reported：
RuntimeError: stack expects each tensor to be equal size, but got [512, 896, 3] at entry 0 and [512, 1152, 3] at entry 1

**Comment by YupingQiu** [comment]
yes, you need to resize your data to the same size, otherwise there is a bug here

**Comment by Chantec** [comment]
> yes, you need to resize your data to the same size, otherwise there is a bug here

If the size is the same, but it's not 512x512, is that okay? My size is 160x160.

**Comment by ziqin-h** [comment]
> ###
> [Nov 18, 2024](#issuecomment-2481995490)

I have the same question, did you find the answer?
